### ISSUE TITLE
UI Fix - Header section height

### ISSUE THREAD

**Issue by tonynguyen111997** [inland-empire-software-development/landing#7]
When changing the view-port's height, only the header section's height is affected. Header section shrinks to fit 60vh. 

**Do you guys think we should leave it this way or would it be better to make the height static?**

------------------------

**View-port Height Small**
![image](https://user-images.githubusercontent.com/36907562/54961298-53cb7780-4f1d-11e9-9efe-ad874d23be27.png)

![image](https://user-images.githubusercontent.com/36907562/54961358-82495280-4f1d-11e9-89be-409180784f34.png)

**View-port Height Large**
![image](https://user-images.githubusercontent.com/36907562/54961374-a2791180-4f1d-11e9-86c0-3d0b7514f32e.png)

![image](https://user-images.githubusercontent.com/36907562/54961383-a7d65c00-4f1d-11e9-9d7f-40c7c761d511.png)

**Comment by lloan** [comment]
This should have a minimum-height set for scenarios like this. At least 600px I believe or enough to contain the contents - which ever is greater.

**Comment by zootechdrum** [comment]
anyone working on this

**Comment by lloan** [comment]
@zootechdrum I don't think so but we're working on a v3 redesign. Let me know if you want to be involved in that.
